### PR TITLE
RFE-2962: Set interface name from node-ip-configuration to configure-ovs

### PR DESCRIPTION
**Pull request by tsorya**
node-ip-configuration will set env file for configure-ovs when it will
set kubelet ip, this will allow to configure ovs with the bridge on the
same interface that kubelet will use

### CODE REVIEW

File: cmd/runtimecfg/node-ip.go [hunk 1]
```diff
@@ -129,27 +131,70 @@ func set(cmd *cobra.Command, args []string) error {
...
 	_, err = cOverride.WriteString(cOverrideContent)
 	if err != nil {
 		return err
 	}
+
+	// Set interface in case it is not br-ex that means that it was already configured and node is rebooting
+	// in that case don't change the previous written interface
+	if chosenAddresses[0].LinkName != "br-ex" {
+		interfaceDir := filepath.Dir(interfaceFile)
+		err = os.MkdirAll(interfaceDir, 0755)
```
Comment (dougsland) [inline cmd/runtimecfg/node-ip.go:145]: should we check if the interfaceDir exists before trying to create?

File: cmd/runtimecfg/node-ip.go [hunk 2]
```diff
@@ -129,27 +131,70 @@ func set(cmd *cobra.Command, args []string) error {
...
+	cOverrideContent := fmt.Sprintf("[Service]\nEnvironment=\"CONTAINER_STREAM_ADDRESS=%s\"\n", chosenAddresses[0].Address)
 	log.Infof("Writing CRI-O service override with content %s", cOverrideContent)
 	_, err = cOverride.WriteString(cOverrideContent)
 	if err != nil {
 		return err
 	}
+
+	// Set interface in case it is not br-ex that means that it was already configured and node is rebooting
+	// in that case don't change the previous written interface
+	if chosenAddresses[0].LinkName != "br-ex" {
```
Comment (dougsland) [inline cmd/runtimecfg/node-ip.go:143]: br-ex seems a good candidate for const ?

**Review by dougsland — COMMENTED** [review]
The patch is draft but it's in my queue for review, looks good but I have few comments.

**Comment by dougsland** [comment]
Similar to https://github.com/openshift/baremetal-runtimecfg/pull/183 ?

**Comment by tsorya** [comment]
it was decided to go with ip address and not setting interface

**Pull request closed without merging**